tools/build.cmd

```diff
@@ -76,13 +76,13 @@ goto args-continue
 
 :arg-enable-dotnet-binding
 set CMAKE_enable_dotnet_binding=ON
-call %current-path%\build_dotnet.cmd
+call %current-path%\build_dotnet.cmd --config %build-config% --platform %build-platform%
 if not !ERRORLEVEL!==0 exit /b !ERRORLEVEL!
```

Comment:
Hey! Sorry, can you reply to my comment here? Want to test Pull Request Review on CodeFlow.